metadatastore/commands.py

```diff
@@ -157,6 +157,7 @@ def insert_run_start(time, scan_id, beamline_id, beamline_config, uid=None,
     if project is None:
         project = ''
 
+    beamline_config = BeamlineConfig.objects(uid=beamline_config).first()
```

Comment:
`.get()` is not a valid method

http://nbviewer.ipython.org/gist/ericdill/ca047302c2c1f1865415